src/layers/createMarkerSVG.ts

```diff
@@ -0,0 +1,34 @@
+const MARKER_PATH =
+    'M172.268 501.67C26.97 291.031 0 269.413 0 192 0 85.961 85.961 0 192 0s192 85.961 192 192c0 77.413-26.97 99.031-172.268 309.67-9.535 13.774-29.93 13.773-39.464 0z'
+const INNER_CIRCLE = 'M192 272c44.183 0 80-35.817 80-80s-35.817-80-80-80-80 35.817-80 80 35.817 80 80 80z'
+
+interface MarkerProps {
+    color: string
+    number?: number
+    size?: number
+}
+
+// todo: this is mostly duplicated from `Marker.tsx`. To use `Marker.tsx` we would probably need to add ol.Overlays, i.e.
+//       create a div for each marker and insert the svg from `Marker.tsx`.
+export function createSvg({ color, number, size = 0 }: MarkerProps) {
+    return `<svg aria-hidden="true" focusable="false" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 384 512" width="${
+        // todo: we do not use width in Marker.tsx, but without this the markers are not shown in Firefox :( (but they are shown in Chrome...)
```

Comment:
* I had to duplicate the svg from `Marker.tsx`, because I needed it as a string to use it as Icon src for the style image of the query points vector layer. The alternative could be using an ol.Overlay for each marker, but then we need to somehow create a div (using useRef?) for each overlay. Considering the issues I ran into here (see below) this might be the better option?

* The marker positions are off. Apparently the `translate` style property doesn't work correctly

* The marker numbers/text is only shown in Chrome. In Firefox it is missing or so small that it is invisible.

* I had to change the colors that are given as hex strings to 'rgb(red, green, blue)'. Otherwise the markers were invisible?!

Reply:
It would be nice to have this mounted as a ```Marker.tsx``` component. This would indeed mean that we would have a wrapper div for each point in a route. One would then create a new rendering context for each point. Maybe this would be too complicated. 

The way this works now, is that ol renders the svg on the canvas as an image [directly](https://github.com/openlayers/openlayers/blob/30d27a1d59113617cc0247dd862cdc763ca76216/src/ol/render/canvas.js#L460). I'd assume that this is the most efficient way to do it. 

Leave it like this, fix the translation and the number?

Reply:
Ok that makes sense. Can you have a go at this?